src/clusterfuzz/_internal/datastore/data_types.py

```diff
@@ -375,6 +375,9 @@ class Testcase(Model):
   # Complete stacktrace.
   crash_stacktrace = ndb.TextProperty(indexed=False)
 
+  # Hotlists associated with the crash.
+  crash_subtypes = ndb.StringProperty(repeated=True)
```

Comment:
Rather than a full blown Testcase property (which is useful when we need to add a filter etc, could we instead use ` testcase.set_metadata` ? 

(Also, I think the comment is wrong :) )

CC @jonathanmetzman for thoughts too.

Reply:
Thanks for the suggestion! I refactored the code to use set_metadata instead of a new Testcase property.